src/App/Pages/Vault/GroupingsPage/GroupingsPage.xaml.cs

```diff
@@ -202,46 +202,53 @@ protected override async void OnDisappearing()
 
         private async void RowSelected(object sender, SelectionChangedEventArgs e)
         {
-            ((ExtendedCollectionView)sender).SelectedItem = null;
-            if (!DoOnce())
+            try
             {
-                return;
-            }
+                ((ExtendedCollectionView)sender).SelectedItem = null;
+                if (!DoOnce())
+                {
+                    return;
+                }
 
-            if (e.CurrentSelection?.FirstOrDefault() is GroupingsPageTOTPListItem totpItem)
-            {
-                await _vm.SelectCipherAsync(totpItem.Cipher);
-                return;
-            }
+                if (e.CurrentSelection?.FirstOrDefault() is GroupingsPageTOTPListItem totpItem)
+                {
+                    await _vm.SelectCipherAsync(totpItem.Cipher);
+                    return;
+                }
 
-            if (!(e.CurrentSelection?.FirstOrDefault() is GroupingsPageListItem item))
-            {
-                return;
-            }
+                if (!(e.CurrentSelection?.FirstOrDefault() is GroupingsPageListItem item))
+                {
+                    return;
+                }
 
-            if (item.IsTrash)
-            {
-                await _vm.SelectTrashAsync();
-            }
-            else if (item.IsTotpCode)
-            {
-                await _vm.SelectTotpCodesAsync();
-            }
-            else if (item.Cipher != null)
-            {
-                await _vm.SelectCipherAsync(item.Cipher);
-            }
-            else if (item.Folder != null)
-            {
-                await _vm.SelectFolderAsync(item.Folder);
-            }
-            else if (item.Collection != null)
-            {
-                await _vm.SelectCollectionAsync(item.Collection);
+                if (item.IsTrash)
+                {
+                    await _vm.SelectTrashAsync();
+                }
+                else if (item.IsTotpCode)
+                {
+                    await _vm.SelectTotpCodesAsync();
+                }
+                else if (item.Cipher != null)
+                {
+                    await _vm.SelectCipherAsync(item.Cipher);
+                }
+                else if (item.Folder != null)
+                {
+                    await _vm.SelectFolderAsync(item.Folder);
+                }
+                else if (item.Collection != null)
+                {
+                    await _vm.SelectCollectionAsync(item.Collection);
+                }
+                else if (item.Type != null)
+                {
+                    await _vm.SelectTypeAsync(item.Type.Value);
+                }
             }
-            else if (item.Type != null)
+            catch (Exception ex)
             {
-                await _vm.SelectTypeAsync(item.Type.Value);
+                LoggerHelper.LogEvenIfCantBeResolved(ex);
             }
```

Comment:
we should also let the user know that something went wrong, if not they might think that the action takes a lot to complete or something like that

Reply:
true, i'll add a generic error dialog.